### PR TITLE
59649: Update text to include the new slider bar for VMs.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/virtualmachines.rst [hunk 1]
```diff
@@ -52,6 +52,15 @@ shows a list of installed virtual machines.
 
 The |ui-options| menu has options for controlling and modifying VMs:
 
+* :guilabel:`Start` boots a VM. Alertnatively start a VM by clicking the
```
Comment (Mrt134) [inline userguide/virtualmachines.rst]:
```suggestion
* :guilabel:`Start` boots a VM. Alternatively, start a VM by clicking the
```

File: userguide/virtualmachines.rst [hunk 2]
```diff
@@ -52,6 +52,15 @@ shows a list of installed virtual machines.
 
 The |ui-options| menu has options for controlling and modifying VMs:
 
+* :guilabel:`Start` boots a VM. Alternatively, start a VM by clicking the
```
Comment (wblock) [inline userguide/virtualmachines.rst]: Please avoid the "Alternatively," halting start to a sentence.  How about
"VMs can also be started by..."

File: userguide/virtualmachines.rst [hunk 3]
```diff
@@ -52,6 +52,15 @@ shows a list of installed virtual machines.
...
+  slide bar on the desired VM. An option is provided to
```
Comment (wblock) [inline userguide/virtualmachines.rst]: s/bar/toggle/

Break paragraph to talk about memory overcommit.

File: userguide/virtualmachines.rst [hunk 4]
```diff
@@ -52,6 +52,15 @@ shows a list of installed virtual machines.
...
+  :guilabel:`Overcommit Memory`. Memory overcommitment allows multiple
+  VMs to be launched when there is not enough free memory for all of
+  them to run at the same time. This option should be used with
+  caution. When active, the VM :guilabel:`State` changes to
```
Comment (wblock) [inline userguide/virtualmachines.rst]: Break paragraph after "caution."

**Pull request merged**